src/api/DataApi.js

```diff
@@ -688,6 +689,55 @@ function updateEntityData(
     });
 }
 
+/**
+ * `POST /data/binary`
+ *
+ * Loads a presigned URL for a particular binary object with the requested content disposition
+ *
+ * @static
+ * @memberof lattice.DATA_API
+ * @param {Object} binaryObjectRequest
+ * @returns {Promise<Object>} - a Promise that resolves the same request structure, with the
+ * content disposition field replaced by a presigned URL for the requested binary object, with the
+ * specified content disposition
+ *
+ * @example
+ * DataApi.loadBinaryProperties(
+ *  {
+ *    "value": {
```

Comment:
`binaryObjectRequest` is the object that `value` points to, so `value` should not exist. Please update.

Reply:
this strangely shaped payload is what the backend expects